### DISCUSSION TITLE
Transparency in model keeps crashing Safari page

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4965] [Q&A]
I have a model and one of its parts contains transparency. The webpage kept crashing while loading ("A problem repeatedly occured...") and I couldn't figure out what seemed to be the issue. After removing parts of the model I realized that the parts that had transparency were causing the issue. When I removed the parts in question the page would load all the way, when I removed everything but the parts with transparency the page would keep crashing even though the model was now below 300kb. 

Did anyone have this kind of issue before? Any suggestions? (Manual conversion to USDZ wont work because the model changes dynamically before entering AR)

I tested on multiple iOS versions and devices and had same results. (iPhone Xr (iOS 17.4.1), iPhone 8 (iOS 16.7.10), iPhone 13 Pro (not sure which OS version but I think 18).
Using the latest model-viewer.min.js version.

**Comment by elalish** [comment]
Yes, we have lots of issues like this because Apple hasn't published a spec of what they allow in USDZ, and every year they seem to introduce breaking changes and stop rendering a bunch of previously working USDZs. Please lobby Apple to support an actual standard like glTF. FYI @AdaRoseCannon.
